connectors/.eslintrc.js

```diff
@@ -41,6 +41,8 @@ module.exports = {
     ],
     "@typescript-eslint/no-unused-vars": "error",
     "@typescript-eslint/return-await": ["error", "in-try-catch"],
+    // TODO(flavien): fix resource pattern
+    "@typescript-eslint/no-unsafe-declaration-merging": "off",
```

Comment:
Not super convinced by this line. Why are we changing it here? It's a new by default rule?

Reply:
Was thinking about it will create exceptions on every files instead of a global rule